src/azure-cli/azure/cli/command_modules/serviceconnector/custom.py

```diff
@@ -182,8 +181,6 @@ def connection_list_configuration(client,
                                    resource_uri=source_id,
                                    linker_name=connection_name)
 
-    decorate_springboot_cosmossql_config(configurations)
```

Comment:
why remove this

Reply:
It is implemented at backend with the new filed of description.